src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -41,20 +40,20 @@ const en = ({ isDivorce, partner, userCase, isJointApplication }: CommonContent)
     } as a sole applicant.
     You will receive an email with information on how to change your application, if they do not apply.`,
   },
+  readMoreChangeToSoleSubHeader: 'If you want to change to a sole application now',
   readMoreChangeToSole: {
     line1: `If you know your ${partner} is not going to confirm the joint application, then you can apply to change to a sole application now.
     You should save and sign out and follow these steps.`,
-    orderedList1FinalOrder: `<a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/952032/d36-eng.pdf"
-    download="Download-Fill-Out-Application-For-Final-Order">Download and fill out an application for a final order</a>. (as a sole applicant)`,
-    orderedList2CertificateOfService: `Give notice to your ${partner} that you are intending to apply for a final order as a sole applicant.
-    You can do this by sending them a draft copy of the application by email or post. You will need to  <a class="govuk-link" href="https://www.gov.uk/government/publications/form-n215-certificate-of-service"
-    download="Download-Fill-Out-Certificate-Of-Service">fill out a 'certificate of service'</a> form to prove to the court that you have given them notice.`,
+    orderedList1FinalOrder: ' (as a sole applicant)',
+    orderedList2CertificateOfServicePart1: `Give notice to your ${partner} that you are intending to apply for a final order as a sole applicant.
+    You can do this by sending them a draft copy of the application by email or post. You will need to `,
+    orderedList2CertificateOfServicePart2: 'form to prove to the court that you have given them notice.',
     orderedList3: `You have to give your ${partner} 14 days to respond, starting from the day they are ‘served’ the application. After 14 days, then send the following documents and evidence to the court:`,
-    bulletPoint1: `The <a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/952032/d36-eng.pdf"
-    download="Download-Fill-Out-Application-For-Final-Order">application for a final order</a> which proves that you have given notice to your ${partner} that you are intending to apply for a final order as a sole applicant.`,
-    bulletPoint2: `The <a class="govuk-link" href="https://www.gov.uk/government/publications/form-n215-certificate-of-service"
-    download="Download-Fill-Out-Certificate-Of-Service"> 'certificate of service'</a> which proves that you have given notice to your ${partner} that you are intending to apply for a final order as a sole applicant.`,
-    postOrEmail:
+    bulletPoint1Part1: 'The',
```

Comment:
these can be an object and have parts like we do in other places.

Reply:
Updated the content the content file to include objects.